api-test/hasura/actions/_handlers/updateEpoch.test.ts

```diff
@@ -19,10 +20,11 @@ const now = DateTime.now();
 const DURATION_IN_DAYS = 3;
 
 beforeEach(async () => {
+  const name = faker.name.firstName();
   address = await getUniqueAddress();
   circle = await createCircle(adminClient);
-  profile = await createProfile(adminClient, { address });
-  await createUser(adminClient, { address, circle_id: circle.id });
+  profile = await createProfile(adminClient, { address, name });
```

Comment:
tests may fail sporadically due to this change. this is because `faker.name.firstName` repeats names after a while.

anyway, you don't need to generate a name here, because of your change to`createProfile`.